ibm/service/satellite/data_source_ibm_satellite_location.go

```diff
@@ -138,6 +138,18 @@ func DataSourceIBMSatelliteLocation() *schema.Resource {
 					},
 				},
 			},
+			"service_subnet": {
+				Type:        schema.TypeString,
```

Comment:
In datasource these should be computed only remove optional and forceNew.
Same for `pod_subnet` also

Reply:
fixed it